packages/salesforcedx-utils-vscode/src/helpers/extensionUris.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as vscode from 'vscode';
+
+function extensionUri(extensionName: string) {
+  const extensionRef = vscode.extensions.getExtension(extensionName);
+
+  if (extensionRef) {
+    return extensionRef.extensionUri;
+  }
+  throw new Error(`Unable to find extension ${extensionName}`);
+}
+
+function join(baseUri: vscode.Uri, relativePath: string) {
+  return vscode.Uri.joinPath(baseUri, relativePath);
+}
+
+export const extensionUris = {
```

Comment:
In the spirit of always handing Uri's the same way created a common helper in utils to handle repeatable tasks.  Think pathUtils but for Uris.